nxdrive/gui/folders_dialog.py

```diff
@@ -3,6 +3,8 @@
 from pathlib import Path
 from typing import TYPE_CHECKING, Dict, List, Optional, Union
 
+from PyQt5.QtGui import QKeyEvent
```

Comment:
See how the import was done in custom_window.py

Reply:
👍